causal-inference-for-the-brave-and-true/04-Graphical-Causal-Models.ipynb

```diff
@@ -344,7 +150,7 @@
     "A \\perp C | B\n",
     "$\n",
     "\n",
-    "Now, let's consider a fork structure. In this case, the same variable causes two other variables down the graph. In this case, the dependence flows backward through the arrows and we have what is called a **backdoor path**. We can close the backdoor path and shut down dependence by conditioning on the common cause."
+    "Now, let's consider a fork structure. The same variable causes two other variables down the graph in this case. In this case, the dependence flows backward through the arrows, and we have a **backdoor path**. We can close the backdoor path and shut down dependence by conditioning on the common cause."
```

Comment:
`in this case` ends this sentence and starts the next. I suggest removing one of the uses

```suggestion
    "Now, let's consider a fork structure. The same variable causes two other variables down the graph. In this case, the dependence flows backward through the arrows, and we have a **backdoor path**. We can close the backdoor path and shut down dependence by conditioning on the common cause."
```